ethereum/spec/src/main/resources/tech/pegasys/teku/spec/config/configs/lukso.yaml

```diff
@@ -114,19 +113,21 @@ MAX_PER_EPOCH_ACTIVATION_CHURN_LIMIT: 8
 # ---------------------------------------------------------------
 # 2**5 * 10**9 (= 32,000,000,000) Gwei
 MAX_EFFECTIVE_BALANCE: 32000000000
+# 2**11 * 10**9 (= 2,048,000,000,000) Gwei
+MAX_EFFECTIVE_BALANCE_ELECTRA: 2048000000000
 # 2**4 * 10**9 (= 16,000,000,000) Gwei
 EJECTION_BALANCE: 16000000000
 
 # Extra consensus configuration (Teku)
 # ---------------------------------------------------------------
 # 10 * 2**20 (= 10485760, 10 MiB)
-MAX_PAYLOAD_SIZE: 10485760
+GOSSIP_MAX_SIZE: 10485760
+# `10 * 2**20` (=10485760, 10 MiB)
+MAX_CHUNK_SIZE: 10485760
```

Comment:
we don't support this since https://github.com/ethereum/consensus-specs/pull/4045